brave-unbreak.txt

```diff
@@ -89,46 +89,53 @@
 @@https://analytics.*/piwik.$~third-party
 ! Fix for Suicide Prevention Lifeline Chat (brave/brave-browser#3492)
 @@||geoplugin.net/javascript.gp$script,xmlhttprequest,domain=suicidepreventionlifeline.org
+! Block additional trackers
+||sp1.nypost.com$third-party
+||sp.nasdaq.com$third-party
+||lesechos.fr/xtcore.js$third-party
+||y8.com/js/sdkloader/outstream.js$third-party
```

Comment:
This whole block is just moved from the bottom above the tagged optional filters.